config/config_test.go

```diff
@@ -0,0 +1,168 @@
+package config
```

Comment:
It's also worth mentioning that I try to build projects up incrementally, so "exploratory coding" should never be more than a few hours, before you go back, clean up, and add tests. I wouldn't recommend doing 6 months of "exploratory coding" and then going back and adding tests :)